Clarify the ability of using Custom Rules to act on truncated payloads (#19105)
cloudflare/cloudflare-docs: src/content/docs/waf/managed-rules/index.mdx

## Important remarks

Cloudflare analyzes the body of incoming requests up to a certain maximum size that varies according to your Cloudflare plan. For Enterprise customers, the maximum body size is 128 KB, while for other plans the limit is lower. This means that the behavior of specific managed rules that analyze request bodies can vary according to your current Cloudflare plan.

> :::note
> If included in your plan, you can use the [HTTP request body fields](/ruleset-engine/rules-language/fields/http-request-body/) such as `http.request.body.truncated` or `http.request.headers.truncated` to create [Custom Rules](/waf/custom-rules/) and apply appropriate actions to requests that have been truncated due to the maximum body size.
> :::

Reviewer: ```suggestion
If included in your plan, you can use [request body fields](/ruleset-engine/rules-language/fields/http-request-body/) such as `http.request.body.truncated` or `http.request.headers.truncated` to create [custom rules](/waf/custom-rules/) that apply appropriate actions to requests that have not been fully analyzed by Cloudflare due to the maximum body size.
```
